machine-learning/Intro To Machine Learning.ipynb

```diff
@@ -0,0 +1,320 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Introduction to ML.NET Concepts\n",
+        "ML.NET gives you the ability to add machine learning to .NET applications, in either online or offline scenarios. With this capability, you can make automatic predictions using the data available to your application. Machine learning applications make use of patterns in the data to make predictions rather than needing to be explicitly programmed.\n",
+        "\n",
+        "Central to ML.NET is a machine learning model. The model specifies the steps needed to transform your input data into a prediction. With ML.NET, you can train a custom model by specifying an algorithm, or you can import pre-trained TensorFlow and ONNX models.\n",
+        "\n",
+        "Once you have a model, you can add it to your application to make the predictions.\n",
+        "\n",
+        "ML.NET runs on Windows, Linux, and macOS using .NET Core, or Windows using .NET Framework. 64 bit is supported on all platforms. 32 bit is supported on Windows, except for TensorFlow, LightGBM, and ONNX-related functionality.\n",
+        "\n",
+        "Examples of the type of predictions that you can make with ML.NET:\n",
+        "\n",
+        "Classification/Categorization\n",
+        "\n",
+        "Automatically divide customer feedback into positive and negative categories\n",
+        "\n",
+        "Regression/Predict continuous values\n",
+        "\n",
+        "Predict the price of houses based on size and location\n",
+        "\n",
+        "Anomaly Detection\n",
+        "\n",
+        "Detect fraudulent banking transactions\n",
+        "\n",
+        "Recommendations\n",
+        "\n",
+        "Suggest products that online shoppers may want to buy, based on their previous purchases\n",
+        "\n",
+        "Time series/sequential data\n",
+        "\n",
+        "Forecast the weather/product sales\n",
+        "\n",
+        "Image classification\n",
+        "\n",
+        "Categorize pathologies in medical images"
```

Comment:
Format these into a table :

For example...
| Task | Example |
| --- | --- | 
| Classification / Categorization |  Automatically divide customer feedback into positive and negative categories |